modules/issue_tracker/php/issue_tracker.class.inc

```diff
@@ -115,7 +115,10 @@ class Issue_Tracker extends \NDB_Menu_Filter_Form
             $assignees[$a_row['UserID']] = $a_row['Real_name'];
         }
 
-        $modules = \Module::getActiveModules($db);
+        $modules_names  = array_map(function(\Module $m): string {
+            return $m->getLongName();
+        },\Module::getActiveModules($db));
+        $modules = array_combine($modules_names, $modules_names);
```

Comment:
This line seems a little.. opaque. Why are you combining `modules_names` with itself?

Reply:
Because the datatable is filtering on the cell content which is the module lang name. Therefor, array combine creates an array where keys and values are the module long name for the dropdown options.

I could do all that in the array_map though.

Do you want me to change it?

Reply:
I think at least add a comment explaining it.